### PR TITLE
[Suggestion] Updating stubs

### PR DESCRIPTION
**Pull request by Mxrcon**
# Updating stubs
Hey @abhi18av I'd like to contribute with your work, and i tried to update the stubs that you've already written adding the echo "" command for easier reading on tower execution. I also updated the script to generate mock data and created sutb_params.yml

## PR content
* Stubs to all the process
* mock data generator script
* stub_params.yml

## Ending

If you have any question, critic or suggestion, feel free to contact me and I'll be on your disposal.

Hope we have a merge! 
Regards, Davi

### CODE REVIEW

File: modules/trimmomatic/trimmomatic.nf [hunk 1]
```diff
@@ -50,6 +50,17 @@ process TRIMMOMATIC {
...
+    PE \                                                                         
+    -threads ${task.cpus} \                                                      
+    -phred33 \                                                                   
+    ${genomeReads[0]} \                                                          
+    ${genomeReads[1]} \                                                          
+    $fq_1_paired \                                                               
+    $fq_1_unpaired \                                                             
+    $fq_2_paired \                                                               
+    $fq_2_unpaired \                                                        
+    LEADING:3 TRAILING:3 SLIDINGWINDOW:4:20 MINLEN:36"
```
Comment (abhi18av) [inline modules/trimmomatic/trimmomatic.nf]: Could you please run the workflow using `-stub -with-tower` and share that with me? I'd like to see the exact output of this command.

Comment (Mxrcon) [reply modules/trimmomatic/trimmomatic.nf]: replied on another PR, but you can check  `cranky_roentgen`  tower execution that I shared with you.

Comment (abhi18av) [reply modules/trimmomatic/trimmomatic.nf]: Cool, I've checked and it works as expected 👍

**Review by abhi18av — APPROVED** [review]
Looks good to merge 💯

**Pull request merged**